src/components/CaseStudies/CaseStudiesConfiguration.tsx

```diff
@@ -0,0 +1,54 @@
+import { t } from '@lingui/macro'
+import { ConfigStat } from './ConfigStat'
+
+export interface CaseStudiesConfigurationProps {
+  cycleNumber: number
+  cycles: string
+  duration?: string
+  payouts: string
+  tokenName: string
+  tokenIssuanceRate: string
+  reservedRate: number
+  issuanceReductionRate: number
+  redemptionRate: number
+  ownerTokenMinting: boolean
+  editDeadline: string
+}
+
+export function CaseStudiesConfiguration({
+  cycleNumber,
+  cycles,
+  duration,
+  payouts,
+  tokenName,
+  tokenIssuanceRate,
+  reservedRate,
+  issuanceReductionRate,
+  redemptionRate,
+  ownerTokenMinting,
+  editDeadline,
+}: CaseStudiesConfigurationProps) {
+  return (
+    <div>
+      <div className="mb-4 font-bold">{t`CYCLE #${cycleNumber}`}</div>
```

Comment:
Probably just use `font-medium` here. We only have font-normal or font-medium for Beatrice.

<img width="532" alt="Screenshot 2023-04-21 at 3 13 10 PM" src="https://user-images.githubusercontent.com/12551741/233549920-42e4cca1-abc9-4f9a-8d71-cdd8cc10b694.png">